File: Level1/x만큼 간격이 있는 n개의 숫자/Solution.java
class Solution {
    public static void main(String[] args){
        int x = 2;
        int n = 5;

        // debug
        long[] ans = solution(x,n);
        for(int i=0; i<ans.length; i++)
            System.out.print(ans[i] + " ");
        System.out.println();
    }

    public static long[] solution(int x, int n) {
        long xx = x;
        long[] answer = new long[n];
        for(int i=0; i<n; i++)
            answer[i] = xx*(i+1);
        return answer;
    }
}

File: Level1/x만큼 간격이 있는 n개의 숫자/solution.cpp
#include <string>
#include <vector>
#include <iostream>

using namespace std;

vector<long long> solution(int x, int n) {
    vector<long long> ans;

    for (int i = 1; i <= n; i++) {
        ans.push_back(x * i);
    }
    return ans;
}

int main() {
    int x = 2;
    int n = 5;
    vector<long long> ans = solution(x, n);

    for (long long a : ans) {
        cout << a << endl;
    }

    return 0;
}